data/conf/hive-site.xml

```diff
@@ -130,6 +130,21 @@
   <description></description>
 </property>
 
+<property>
+  <name>yarn.app.mapreduce.am.command-opts</name>
```

Comment:
MR is deprecated, so I am not sure if we need this anymore

Reply:
MR is deprecated but not completely removed yet as per [HIVE-27972](https://issues.apache.org/jira/browse/HIVE-27972) & multiple qtests (qtest drivers) still run on MR. 
So, we do need these configs for MR to ensure compatibility with jdk-17.